Index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Style.css">
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+</head>
+<body>
+
+ <header class="top-bar">
+  <h2 class="logoname">My Logo</h2>
+  <nav >
+    <img src="images\Union.png" alt="logo">
+  </nav>
+</header>
+ <section class="section-data">
+    <h2 class="myname">I’m Mariane
+      Glad to see you!</h2>
+    <h3 class="namedetail">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h3>
+    <div><h3 class="connect">LET’S CONNECT</h3></div>
+    <div class="socialmedia">
+      <img class="twitter" src="images\twitter.png" alt="twitter"></div>
+      <img class="dot" src="images\dot.png" alt="dot"></div>
+      <img class="i" src="images\i.png" alt="i"></div>
+      <img class="n" src="images\n.png" alt="n"></div>
+      <img class="m" src="images\M.png" alt="m"></div>
+      <img class="g" src="images\g.png" alt="g"></div>
+      <img class="v" src="images\v.png" alt="v"></div>
```

Comment:
Kindly give the proper margin to this section. Currently, it is aligned with the bottom of the page which doesn't match the Figma template of your choosing.  Kindly fix this issue.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/219358904-5da3746b-82b0-4c11-a5f7-0447ce6cb087.png) | ![image](https://user-images.githubusercontent.com/33268927/219359262-3b15f734-8fc1-4298-a73f-cbe65d6b1a03.png) |